package.json

```diff
@@ -113,6 +113,7 @@
     "react-scripts": "4.0.1",
     "react-spring": "^8.0.27",
     "react-transition-group": "^4.4.1",
+    "react-use-measure": "^2.0.4",
```

Comment:
I believe you were using that in one of the first drafts for the sheet, that's a cool util, we might use that in other areas in the future so 👌

Reply:
Yes thats right, very useful 😎

Reply:
Yeah it's on me for removing it, I may have tried to oversimplify things 😅